geometric_data/ocean/transect/Denmark_Strait/transect.geojson

```diff
@@ -5,65 +5,64 @@
             "type": "Feature",
             "properties": {
                 "name": "Denmark Strait",
-                "tags": "standard_transport_sections;arctic_sections",
+                "tags": "standard_transport_sections;arctic_transport_sections",
                 "object": "transect",
                 "component": "ocean",
-                "author": "Milena Veneziani",
-                "history": "03/20/21 : milena : hand drawn in geojson.io;"
+                "author": "Milena Veneziani"
             },
             "geometry": {
                 "type": "LineString",
                 "coordinates": [
                     [
-                        -24.06005859375,
-                         65.52206835254366
+                        -24.060059,
+                        65.522068
```

Comment:
This rounding is expected.  See:
https://github.com/MPAS-Dev/geometric_features/blob/master/geometric_features/feature_collection.py#L442
and
https://github.com/MPAS-Dev/geometric_features/blob/master/geometric_features/feature_collection.py#L739-L744

We round to 6 digits so reading, combining, and splitting features is less likely to change them in ways that lead to undesirable changes.  My guess is that these features were not split in the usual way when they were first added.